indico/modules/events/abstracts/templates/management/abstract.html

```diff
@@ -1,6 +1,180 @@
 {% extends 'events/abstracts/management/_base.html' %}
-{% from 'events/abstracts/_abstract.html' import render_abstract_public %}
+{% from 'events/abstracts/_abstract.html' import render_abstract_public, render_abstract_comments, render_edited_hint %}
 
 {% block content %}
     {{ render_abstract_public(abstract, management=true) }}
+    <div id="abstract-private" class="i-timeline">
+        <div class="i-timetable-connect-down"></div>
+        <div class="i-timeline-separator"></div>
+        {{ render_abstract_comments(abstract) }}
+        {{ _render_abstract_reviews(abstract) }}
+        <div class="i-timeline-separator"></div>
+        <div class="i-timetable-connect-up"></div>
+        {{ _render_decision_box(abstract) }}
+    </div>
 {% endblock %}
+
+{% macro _render_abstract_reviews(abstract) %}
+    <div id="abstract-reviews" class="i-timeline">
+        {% set reviews = abstract.reviews|selectattr('track', 'equalto', none)|list %}
+        {% if reviews %}
+            <h3>No track</h3>
+            {% for review in reviews %}
+                {{ _render_review(review) }}
+            {% endfor %}
+        {% endif %}
+        {% for track in abstract.reviewed_for_tracks %}
+            {% set reviews = abstract.reviews|selectattr('track', 'equalto', track)|list %}
+            {% if reviews %}
+                <h3>
+                    {% trans title=track.title -%}
+                        Track: {{ title }}
+                    {%- endtrans %}
+                </h3>
+                {% for review in reviews %}
+                    {{ _render_review(review) }}
+                {% endfor %}
+            {% endif %}
+        {% else %}
+        {% endfor %}
+    </div>
+{% endmacro %}
+
+{% macro _render_review(review) %}
+    <div class="i-timeline-item">
+        <div class="i-timeline-item-label action abstract-comment-pic"></div>
+        <div class="i-timeline-item-box header-indicator">
+            <div class="i-box-header flex">
+                <div class="stretch">
+                    {% trans name=review.user.name, date=review.created_dt|format_human_date -%}
+                        <strong>{{ name }}</strong> reviewed on {{ date }}
+                    {%- endtrans %}
+                    {% if review.modified_dt %}
+                        {{ render_edited_hint(comment.modified_dt) }}
+                    {% endif %}
+                </div>
+                <div class="abstract-comment-badges">
+                    <span>
+                        {% trans %}Proposes to{% endtrans %}
```

Comment:
similar solution as in https://github.com/indico/indico/pull/2611/files#r81586082 maybe?